library/src/jvmMain/kotlin/com/thumbtack/kotlin/test/GenerateTestObject.kt

```diff
@@ -66,20 +68,24 @@ import kotlin.reflect.javaType
  *     specify a consistent value for this field.
  * @param useNullForNullableFields If true, then all fields that are nullable will be set to null,
  *     unless specified in the overrides map.
+ * @param collectionSize If specified, supported collection types will be set to that size rather
+ *     than
```

Comment:
I think the sentence is incomplete?

Reply:
I distinctly remember getting distracted with a meeting while writing  this, and I guess not finishing the comment when pushing